dll/cpl/desk/screensaver.c

```diff
@@ -594,6 +667,40 @@ OnInitDialog(HWND hwndDlg, PDATA pData)
         return FALSE;
     }
 
+    wc.lpfnWndProc = DefWindowProc;
+    wc.hInstance = hApplet;
+    wc.hCursor = NULL;
+    wc.hbrBackground = (HBRUSH)GetStockObject(BLACK_BRUSH);
+    wc.lpszClassName = szPreviewWndClass;
+
+    if (RegisterClass(&wc))
+    {
+        HWND hParent = GetDlgItem(hwndDlg, IDC_SCREENS_PREVIEW);
+        HWND hChild;
+
+        if (hParent != INVALID_HANDLE_VALUE)
```

Comment:
I think `INVALID_HANDLE_VALUE` is provided for kernel handles.

```suggestion
        if (hParent != NULL)
```

Reply:
Fixed.